systems/plants/collision/Model.cpp

```diff
@@ -24,6 +24,15 @@ namespace DrakeCollision
       return nullptr;
     }
   }
+  
+  void Model::getTerrainPoints(ElementId id0, Eigen::Matrix4Xd &terrain_points) {
+    auto element_iter = elements.find(id0);
+    if ( element_iter != elements.end()) {
+      element_iter->second->getPoints(terrain_points);
+    } else {
+      terrain_points = Eigen::Matrix4Xd();
```

Comment:
From the Eigen documentation (concerning the default Matrix constructor): "For dynamic-size matrices, creates an empty matrix of size 0. Does not allocate any array. Such a matrix is called a null matrix. This constructor is the unique way to create null matrices: resizing a matrix to 0 is not supported."

As I understand it, this is what we want, and we don't have to hard-code the 4 rows.